scwidgets/widgets.py

```diff
@@ -0,0 +1,230 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
So users will use this as `import scwidgets`? Have you considered other names such as `scipp_widgets` (to match the package name better)? Not sure what is the best solution here.